web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/TomeEventSubscriber.php

```diff
@@ -91,14 +91,16 @@ public function excludeDirectories(CollectPathsEvent $event) {
           $paths[$path] = $metadata;
         }
       }
-
+      // Get the base path without trailing slash if we're exporting to a sub-directory
+      $base_path = rtrim(trim(base_path()), '/');
       foreach ($excluded_directories as $excluded_directory_path) {
         $excluded_directory = $excluded_directory_path . '/';
         if (($path == $excluded_directory_path) ||
```

Comment:
I wound up with two unexpected paths in my export: 
```
.../static/export/jsonapi/index.html
.../static/export/paragraphs_entity_embed/autocomplete/index.html
```
I suspect, but did not verify, that this clause should be expanded like the one on line 103 to catch those.

Reply:
Yes, I missed testing the directory paths. Latest commit fixes this.